### ISSUE TITLE
gap elements with unit but no quantity or extent

### ISSUE THREAD

**Issue by emylonas** [Brown-University-Library/iip-texts#129]
```
kafr0011.xml: <gap reason="lost" unit="character"/>
zoor0388.xml: <gap reason="lost" unit="character" quantity="2"/>Α<gap reason="lost" unit="character"/>
mger0088.xml: <gap reason="lost" unit="character"/>
khir0001.xml: <gap reason="illegible" unit="character"/>άνου <expan>
idum0112.xml: <gap reason="lost" unit="character"/>
hamm0049.xml: <gap reason="illegible" unit="metric"/>
idum0027.xml: <gap reason="illegible" unit="character"/>
zoor0449.xml: <gap reason="lost" unit="line"/>

```

**Comment by GaiaLembi** [comment]
@emylonas extent="unknown" should be added to all: could you implement a global change on these or shall I add it manually?

**Comment by emylonas** [comment]
I'll do it

**Comment by emylonas** [comment]
closed with 1dee02f56
